Gemfile.lock

```diff
@@ -245,6 +245,7 @@ GEM
 
 PLATFORMS
   arm64-darwin-22
+  x86_64-darwin-21
```

Comment:
interesting - what platform are you on? do you still run your terminal from rosetta?